exosphere-shared/src/docker-helper.ls

```diff
@@ -50,6 +51,24 @@ class DockerHelper
       done null, map((.RepoTags |> head |> (.split ':') |> head), images)
 
 
+  @pull-image = ({image}, done) ->
+    docker.pull image, (err, stream) ->
+      console.log "Downloading docker image for '#{image}'..."
+      docker.modem.follow-progress stream, (err, output) ->
+        | err       => done err
+        | otherwise => console.log "'#{image}' download complete"; done!
+
+
+  # Runs cat on file in Docker container to print its content
+  @cat-file = ({image, file-name}, done) ->
+    DockerHelper.pull-image {image}, ->
+      stdout-stream = new stream.PassThrough
+      text-stream-search = new TextStreamSearch stdout-stream
```

Comment:
This seems like an odd instance to use a TextStreamSearch. I would think something like [text-stream-accumulator](https://github.com/Originate/node-text-stream-accumulator) makes more sense here